zram-generator.conf.example

```diff
@@ -19,6 +19,14 @@ host-memory-limit = 9048
 # The default is "min(ram / 2, 4096)".
 zram-size = min(ram / 10, 2048)
 
+# The maximum size of resident memory of zram device, as function of 
```

Comment:
"The maximum memory resident for this zram device, as a function of MemTotal, both in MB." Also format this like the sexion above :v